lib/doorkeeper/models/concerns/revocable.rb

```diff
@@ -8,6 +8,29 @@ def revoke(clock = Time)
       def revoked?
         !!(revoked_at && revoked_at <= Time.now)
       end
+
+      def revoke_in(time)
+        update_attribute :revoked_at, Time.now + time
+      end
+
+      def revoke_previous_refresh_token!
+        old_refresh_token = AccessToken.by_refresh_token(previous_refresh_token)
+        if old_refresh_token && !old_refresh_token.revoked?
+          if configuration.refresh_token_revoked_in.zero?
```

Comment:
We don't need to put this conditional, because both branches will end up having the same behavior.